2023-12-31 14:12:55,977 INFO    MainThread:8232 [wandb_setup.py:_flush():76] Current SDK version is 0.16.1
2023-12-31 14:12:55,977 INFO    MainThread:8232 [wandb_setup.py:_flush():76] Configure stats pid to 8232
2023-12-31 14:12:55,977 INFO    MainThread:8232 [wandb_setup.py:_flush():76] Loading settings from C:\Users\14055\.config\wandb\settings
2023-12-31 14:12:55,977 INFO    MainThread:8232 [wandb_setup.py:_flush():76] Loading settings from F:\Everything\New_drone\wandb\settings
2023-12-31 14:12:55,977 INFO    MainThread:8232 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-12-31 14:12:55,977 INFO    MainThread:8232 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-12-31 14:12:55,977 INFO    MainThread:8232 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'loading_main.py', 'program_abspath': 'F:\\Everything\\New_drone\\loading_main.py', 'program': 'F:\\Everything\\New_drone\\loading_main.py'}
2023-12-31 14:12:55,977 INFO    MainThread:8232 [wandb_init.py:_log_setup():524] Logging user logs to F:\Everything\New_drone\wandb\run-20231231_141255-0ihrguj7\logs\debug.log
2023-12-31 14:12:55,977 INFO    MainThread:8232 [wandb_init.py:_log_setup():525] Logging internal logs to F:\Everything\New_drone\wandb\run-20231231_141255-0ihrguj7\logs\debug-internal.log
2023-12-31 14:12:55,977 INFO    MainThread:8232 [wandb_init.py:init():564] calling init triggers
2023-12-31 14:12:55,977 INFO    MainThread:8232 [wandb_init.py:init():571] wandb.init called with sweep_config: {}
config: {'speed': 0.1}
2023-12-31 14:12:55,977 INFO    MainThread:8232 [wandb_init.py:init():614] starting backend
2023-12-31 14:12:55,977 INFO    MainThread:8232 [wandb_init.py:init():618] setting up manager
2023-12-31 14:12:55,978 INFO    MainThread:8232 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn, using: spawn
2023-12-31 14:12:55,980 INFO    MainThread:8232 [wandb_init.py:init():624] backend started and connected
2023-12-31 14:12:55,982 INFO    MainThread:8232 [wandb_init.py:init():716] updated telemetry
2023-12-31 14:12:56,008 INFO    MainThread:8232 [wandb_init.py:init():749] communicating run to backend with 90.0 second timeout
2023-12-31 14:12:56,258 INFO    MainThread:8232 [wandb_run.py:_on_init():2254] communicating current version
2023-12-31 14:12:56,475 INFO    MainThread:8232 [wandb_run.py:_on_init():2263] got version response 
2023-12-31 14:12:56,475 INFO    MainThread:8232 [wandb_init.py:init():800] starting run threads in backend
2023-12-31 14:12:59,669 WARNING MsgRouterThr:8232 [router.py:message_loop():77] message_loop has been closed
2023-12-31 14:13:01,501 WARNING MainThread:8232 [wandb_init.py:init():1192] interrupted
Traceback (most recent call last):
  File "C:\Users\14055\AppData\Local\Programs\Python\Python39\lib\site-packages\wandb\sdk\wandb_init.py", line 1170, in init
    run = wi.init()
  File "C:\Users\14055\AppData\Local\Programs\Python\Python39\lib\site-packages\wandb\sdk\wandb_init.py", line 811, in init
    run_start_result = run_start_handle.wait(timeout=30)
  File "C:\Users\14055\AppData\Local\Programs\Python\Python39\lib\site-packages\wandb\sdk\lib\mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
  File "C:\Users\14055\AppData\Local\Programs\Python\Python39\lib\site-packages\wandb\sdk\lib\mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
  File "C:\Users\14055\AppData\Local\Programs\Python\Python39\lib\site-packages\wandb\sdk\lib\mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
  File "C:\Users\14055\AppData\Local\Programs\Python\Python39\lib\threading.py", line 574, in wait
    signaled = self._cond.wait(timeout)
  File "C:\Users\14055\AppData\Local\Programs\Python\Python39\lib\threading.py", line 316, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
